<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>파일명 일괄변경</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../MyExplorer/img/rename.png</normaloff>../../MyExplorer/img/rename.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>단축키 이름 :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="txtTitle"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>단축키 종류 :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboScDiv">
        <item>
         <property name="text">
          <string>1. Run Program</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2. Clipboard</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="txtDoActions"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>단축키 지정 :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QCheckBox" name="chkAlt">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>5</y>
           <width>45</width>
           <height>19</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Alt</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkCtrl">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>5</y>
           <width>48</width>
           <height>19</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Ctrl</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkShift">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>5</y>
           <width>56</width>
           <height>19</height>
          </rect>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Shift</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>5</y>
           <width>50</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>단축키:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtShortCut">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>5</y>
           <width>30</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="btnQuery">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>3</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>조회</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnSave">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>4</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>저장</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnDelete">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>5</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>삭제</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>10</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Seq : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtSeq">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>8</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>100</number>
         </property>
        </widget>
        <widget class="QPushButton" name="btnNextSeq">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>5</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>다음 순번</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblFiles">
     <column>
      <property name="text">
       <string>Seq</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>단축키 이름</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>단축키 종류</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>doAction</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Alt</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ctrl</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Shift</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>단축키</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
